### PR TITLE
how to run toolbelt locally

### PR DESCRIPTION
**Pull request by smidhonza**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,6 +53,15 @@ Now you should be able to use the toolbelt anywhere on your system:
 blackjack --help
 ```
 
+## Or if you just want to try it locally without installing globally
+there is a way how to run it locally within your project's root folder.
+
+1. install blackjack-toolbelt as a dependency by pasting `"blackjack-toolbelt": "github:sky-uk/blackjack-toolbelt",` into your package.json under dependencies
```
Comment (romidane) [inline README.md]: blackjack-toolbelt is no longer required by components

**Comment by romidane** [comment]
I don't know if these instruction are usefull as the current README explains quite well how to get blackjack-toolbelt up and running locally

**Comment by joshnesbitt** [comment]
I think we might want to discourage people from using the toolbelt this way. Was there a reason you think this is a better approach?

**Comment by smidhonza** [comment]
oh sorry, I just saw it in blackjack-forms and did not wanted to install blackjack-toolbelt globally. I'll remove toolbelt from blackjack-form, and please ignore this pull request then, sorry again

**Pull request closed without merging**